org.lflang/src/org/lflang/ast/ToLf.java

```diff
@@ -831,21 +843,36 @@ public MalleableString caseInitializer(Initializer object) {
     return initializer(object, false);
   }
 
+  private boolean shouldOutputAsAssignment(Initializer init) {
```

Comment:
The name here seems a bit strange. What exactly does this do? Decide if () assignment should be replaced with = assignment? Maybe a short description would be helpful